ps_emailsubscription.php

```diff
@@ -1274,7 +1274,7 @@ public function hookActionExportGDPRData($customer)
             if ($res = Db::getInstance()->ExecuteS($sql)) {
                 return json_encode($res);
             }
-            return json_encode($this->l('Newsletter subscription: Unable to export customer using email.'));
+            return json_encode($this->trans('Newsletter subscription: Unable to export customer using email.', array(), 'Modules.Emailsubscription.Admin'));
```

Comment:
Same here!

Reply:
Checked with Alex Darmon, please turn the current wording into _Newsletter subscription: no email to export, this customer has not registered._